content/400-reference/200-api-reference/200-command-reference.mdx

```diff
@@ -379,32 +379,82 @@ Introspection does not currently support the [MongoDB connector](../../../concep
 
```

Comment:
Let's split this into two different admonitions:
```suggestion
</Admonition>

<Admonition type="warning">
```

Reply:
Actually, the MongoDB warning also belongs with `db pull` now so this will only have the deprecation warning left.

Reply:
Thats fair